src/sentry/utils/performance_issues/detectors/io_main_thread_detector.py

```diff
@@ -118,7 +118,6 @@ def is_detector_enabled(cls) -> bool:
 
     def init(self):
         super().init()
-        self._prepare_deobfuscation()
```

Comment:
looks like you can remove this init function now but should this have something like `self.mapper = None` to initialize it to avoid a potential attribute error?

Reply:
mapper is init in the base class so shoudl be fine :+1: but i'll clean this up